src/elements/core/mixins/panel-common.scss

```diff
@@ -100,6 +105,15 @@
   overflow: hidden;
 }
 
+slot[name='suffix'],
```

Comment:
I'm unsure wether we should define font styles for these slots. Originally we wanted to style as less as possible (but I also see that coloring was made). @kyubisation could you maybe give your opinion?

Reply:
Briefly discussed with @jeripeierSBB.
Avoid adding new styles as we want to refactor this for 2.0 anyway.
Is that ok?

Reply:
Yes, css and stories updated.